common-rest/src/main/java/org/uniprot/api/rest/validation/error/ResponseExceptionHandler.java

```diff
@@ -244,7 +244,7 @@ private ErrorInfo() {
             this("", Collections.emptyList());
         }
```

Comment:
This comment is not related to your changes.
Please rename the package name support_data without underscore.